staging/src/k8s.io/cli-runtime/pkg/genericclioptions/command_headers.go

```diff
@@ -33,8 +34,9 @@ const (
 // round tripper to add Request headers before delegation. Implements
 // the go standard library "http.RoundTripper" interface.
 type CommandHeaderRoundTripper struct {
-	Delegate http.RoundTripper
-	Headers  map[string]string
+	Delegate    http.RoundTripper
+	Headers     map[string]string
+	SkipHeaders *atomic.Bool
```

Comment:
do we really want to skip this, or do we want to error if both features are requested at the same time?